signalfx-org-metrics/metrics.yaml

```diff
@@ -873,10 +883,24 @@ sf.org.numDatapointsDroppedExceededQuotaByToken:
   metric_type: counter
   title: sf.org.numDatapointsDroppedExceededQuotaByToken
 
+sf.org.numDatapointsDroppedInvalid:
+  brief: Number of data points dropped because they didn't follow documented guidelines for data points.
+  description: |
+    Number of data points dropped because they didn't follow documented guidelines for data points. For example, the metric name was too long, the metric name included unsupported characters, or the data point contained no values.
+
+        * Dimension(s):  `orgId`
+        * Data resolution: 10 seconds
+  metric_type: counter
+  title: sf.org.numDatapointsDroppedInvalid
+
 sf.org.numDatapointsDroppedInTimeout:
-  brief: Infrastructure Monitoring internal metric
-  description: Reserved for Infrastructure Monitoring internal use only.
-  metric_type: null
+  brief: Number data points Observability Cloud didn't attempt to create because your account was throttled or limited in the previous few seconds.
+  description: |
+    Number data points Observability Cloud didn't attempt to create because your account was throttled or limited in the previous few seconds and creation was very unlikely to succeed.
```

Comment:
```suggestion
    Number of data points Observability Cloud didn't attempt to create because your account was throttled or limited in the previous few seconds and creation was very unlikely to succeed.
```